sagemaker-python-sdk/scikit_learn_model_registry_batch_transform/scikit_learn_model_registry_batch_transform.ipynb

```diff
@@ -0,0 +1,671 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    data = fetch_california_housing()
</pre><p>Is it possible to use the hosted version <code>s3://sagemaker-sample-files/datasets/tabular/california_housing/cal_housing.tgz</code>?</p><p>See this notebook for similar use:</p><p><a href="https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-pipelines/tabular/train-register-deploy-pipeline-model/train%20register%20and%20deploy%20a%20pipeline%20model.ipynb" target="_blank">https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-pipelines/tabular/train-register-deploy-pipeline-model/train%20register%20and%20deploy%20a%20pipeline%20model.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3086/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done.</p>